omniduct/databases/base.py

```diff
@@ -385,19 +388,19 @@ def _table_props(self, table, **kwargs):
     def _register_magics(self, base_name):
         from IPython.core.magic import register_line_magic, register_cell_magic, register_line_cell_magic
 
-        def statement_executor_magic(executor, statement, variable=None, show='head', transpose=False, template=True, template_context=None, **kwargs):
+        def statement_executor_magic(executor, statement, variable=None, show='head', transpose=False, template=True, context=None, **kwargs):
```

Comment:
I used `template_context` here because there is the option to disable and enabled templating. Using `template` and `template_context` links these two options together. I can see the benefits of having a more consistent API though, especially since most users will assume templating is enabled anyway. LGTM.